engine/src/gfx/matrix.h

```diff
@@ -355,8 +360,8 @@ inline void InvertMatrix( Matrix &o, const Matrix &trans )
 
 inline void Rotate( Matrix &tmp, const Vector &axis, float angle )
 {
-    double c = cos( angle );
-    double s = sin( angle );
+    float c = cos( angle );
+    float s = sin( angle );
```

Comment:
Okay, so this means we don't lose precision on the cos/sin of angle.
Question: do this mean we end up losing precision in M() when processing Matrix and Vector data? Is any of that data stored as `double`? or is it it all `float`?

Reply:
@BenjamenMeyer I just checked, and all of the Matrix and Vector data is stored as `float`.